data/sql/updates/pending_db_world/rev_1714664171505149429.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `spell_target_position` WHERE `ID` = 21128;
+INSERT INTO `spell_target_position`
+(`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`)
+VALUES
+(21128, 0, 349, 386.27, 33.4144, -130.934, 0, 52237);
```

Comment:
I might be wrong, but I don't see a problem with sticking all of these on one line.